packages/mobile/test/schemas.ts

```diff
@@ -1035,6 +1036,18 @@ export const v38Schema = {
   },
 }
 
+export const v39Schema = {
+  ...v38Schema,
+  _persist: {
+    ...v38Schema._persist,
+    version: 38,
```

Comment:
This must be changed to 39.
There's a test around this, but you found a loophole 😅

Reply:
done! :) good catch